src/main/java/org/openjdk/jextract/impl/TreeMaker.java

```diff
@@ -323,11 +323,7 @@ final Type.Declared recordDeclaration(Cursor parent, Cursor recordCursor) {
 
     public Declaration.Scoped createEnum(Cursor c) {
         List<Declaration> decls = new ArrayList<>();
-        c.forEach(child -> {
-            if (!child.isBitField() || (child.getBitFieldWidth() != 0 && !child.spelling().isEmpty())) {
-                decls.add(createTree(child));
-            }
-        });
+        c.forEach(child -> decls.add(createTree(child)));
```

Comment:
Not sure how we ended up with the current code, but an enum can not contain any bitfields AFAIK.

Reply:
I think the code was filtering members in the same way when creating a new scoped (struct/union/enum). It seems like some code got inlined here, but yes, not too sure what bitfields have got to do with enums.